ReleaseProcessAutomation/Steps/PipelineSteps/ContinueReleasePatchStep.cs

```diff
@@ -92,9 +92,28 @@ public void Execute (SemanticVersion nextVersion, bool noPush, bool onMaster)
 
     GitClient.Checkout(mergeTargetBranchName);
 
+    if (!onMaster)
+    {
+      CreateNewSupportBranch(nextVersion);
```

Comment:
Why do we not do this on master, too?

Reply:
Because the only branch releasing to master should be ones coming from the develop branch, where we do not require support branches

Reply:
Because I forgot that when we release from develop we might need a new support branch on the other side ;)

Reply:
Except maybe in this case we do not want this behaviour, as releasing a patch to master is supported but not intended.  We do however require this functionality in other steps releasing to master.

Reply:
Better create one and delete than needing to do all this manually (optimize for the frequent case)

Reply:
Ok, will do!